src/NuGet.Clients/NuGet.PackageManagement.UI/Resources.resx

```diff
@@ -260,21 +260,25 @@
   </data>
   <data name="Button_Preview" xml:space="preserve">
     <value>Preview</value>
-  </data>
+  </data>`
   <data name="WindowTitle_FileConflict" xml:space="preserve">
     <value>File Conflict</value>
   </data>
   <data name="Button_No" xml:space="preserve">
-    <value>No</value>
+    <value>_No</value>
+    <comment>Button_Yes, Button_No, Button_YesToAll, Button_NoToAll should have underscore in different letters. See https://docs.microsoft.com/windows/uwp/design/input/access-keys#choose-access-keys</comment>
```

Comment:
nit: this URL redirects to `https://docs.microsoft.com/en-us/windows/apps/design/input/access-keys#choose-access-keys`

Reply:
That's correct, because MSDN docs redirect to the translated page in the language specified in the client request. It's a good practice to remove language path in MSDN docs URIs to let the user decide language.

Reply:
No, I'm pointing out that it redirected from a UWP specific url to a generic Windows url.

Reply:
Updated URL's